bookkeeper-common-allocator/pom.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
It's to include `bookkeeper-common-allocator` in `pulsar-client` that currently doesn't depend on `bookkeeper-common`, because it would pull in another set of dependencies.